_posts/2017-02-29-upgrade-to-babel-7.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: post
+title:  "Upgrade to Babel 7 (WIP)"
+author: Sven SAULEAU, Henry Zhu
+date:   2017-02-29 00:00:00
+categories: announcements
+share_text: "Upgrade to Babel 7"
+third_party_js:
+- https://platform.twitter.com/widgets.js
+custom_js_with_timestamps:
+- docs.js
+---
+
+Babel 7 is out! Refer users this document when upgrading to Babel 7.
+
+Because not every breaking change will affect every project, we've sorted the sections by the likelihood of a change breaking tests when upgrading.
+
+## All of Babel
+
+> Support for Node.js 0.10 and 0.12 has been dropped [#5025](https://github.com/babel/babel/pull/5025), [#5041](https://github.com/babel/babel/pull/5041), [#5186](https://github.com/babel/babel/pull/5186) ![high](https://img.shields.io/badge/level%20of%20awesome%3F-high-red.svg)
+
+We highly encourage you to use a newer version of Node.js (LTS v4, LTS v6) since the previous versions are not maintained.
+See [nodejs/LTS](https://github.com/nodejs/LTS) for more information.
+
+> This just means Babel *itself* won't run on older versions of Node. It can still *output* code that runs on old Node. 
+
+## babel-preset-stage-3
+
+> Remove Stage 4 plugins from Stage 3 [#5126](https://github.com/babel/babel/pull/5126) ![high](https://img.shields.io/badge/risk%20of%20breakage%3F-high-red.svg)
+
+These plugins were moved into their yearly presets after moving to Stage 4:
+
+`babel-plugin-syntax-trailing-function-commas` (babel-preset-es2017)
```

Comment:
babel-preset-es2017 ➡️ `babel-preset-es2017`

Also for the presets listed below.